Src/PerformanceCollector/Filtering/CollectionConfigurationAccumulator.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Represents the operationalized metric-related component of the QuickPulse accumulator.
```

Comment:
Changed to
"Represents the part of the QuickPulse accumulator which holds operationalized metric data."